hll.proto

```diff
@@ -0,0 +1,14 @@
+package hll;
+
+message hllPb {
```

Comment:
nope. I didn't have a particular name in mind so I went with `HllPb` . any other ideas?